vignettes/function-flow.Rmd

```diff
@@ -97,83 +97,218 @@ library(dplyr)
 library(osdc)
 ```
 
-#### HbA1c tests above the diagnosis cut-off value (48 mmol/mol or 6.5%)
-
-The function `include_hba1c()` uses `lab_forsker` as the input data to
-extract all events of HbA1c tests above the diagnosis cut-off value.
+#### Hospital diagnoses
 
-Since the HbA1c diagnosis cut-off value depends on the kind of test that is
-used, the inclusion event is defined as follows:
+**Joining LPR2 and LPR3 data**
 
--   For HbA1c IFCC (NPU03835), we include values \>= 6.5 %.
--   For HbA1c DCCT (NPU27300), we include values \>= 48 mmol/mol.
+The helper functions `join_lpr2()` and `join_lpr3()` join records of
+diagnoses to administrative information in LPR2-formatted and
+LPR3-formatted data, respectively.
 
-```{r, echo=FALSE}
-algorithm |> 
-	filter(name=="hba1c") |>
-	knitr::kable(caption = "Algorithm used in the implementation for including HbA1c.")
-```
+`join_lpr2()` takes `lpr_diag` and `lpr_adm` as inputs, filters to the
+necessary diagnoses (`c_diag` starting with "DO", "DZ3", "DE1[0-4]",
+"249", or "250"), joins the required information by record number
+(`recnum`), and outputs a `data.frame` with the following variables:
 
-#### Hospital diagnosis of diabetes
+-   identifier variable (`pnr`)
+-   date (`d_inddto`)
+-   department specialty (`c_spec`)
```

Comment:
Is there any filtering we want to do beforehand for this as well?

Reply:
Yes. Fixed in latest commit.